### DISCUSSION TITLE
ToonSquid slow to open/load?

### DISCUSSION THREAD

**Discussion by judgeworks** [keiwando/toonsquid#356] [General]
Hi- have been using ToonSquid past 4 months and love it. 

noticed the last few days that it’s been loading/opening much slower than it did up until a few days ago. Also when backing up sometimes I am getting same delay, the loading/saving slider seems to stop at the 25% level on slider (or there shouts), but then eventually it loads and/or saves.

Normally ToonSquid opens And saves/backups in a second. Now it takes about 30 seconds.

is there something I need to do to remedy? I checked my iPad storage and I have loads of space. My ToonSquid files take up 57GB—could that be the problem? I have an M1 iPad. I keep ToonSquid up to date with updates.

Thanks for any help.

**Comment by keiwando** [comment]
Hi! Without seeing the specific project it's unfortunately pretty hard to tell what might be going on here. 

One thing to note is that there were some significant changes to parts of the project file format which improves autosave speeds and thereby prevents frequent freezes that some users were experiencing in large projects (especially when there were lots of vector brush strokes in a clip). That change was part of 1.9.1.

Projects that were saved in older versions run through an automatic one-time migration to the new format, which only happens the first time that you open the project again. That first load will therefore take longer than it did before, but again, that's just a one-time delay per project. There were also other optimizations made that speed up the saving process when you close the current project. The save time for backups should not have been noticeably affected by any of these changes (a 30x slowdown that you mention would definitely be unexpected).
The slider stopping at about 25% for a while before continuing definitely sounds like that automatic migration, but that is never performed when creating a backup, so I'm a little confused about you mentioning this in the context of backups.

If you have a specific project with which you can consistently reproduce these slowdowns, please feel free to send me the project file to contact@toonsquid.com and I will be happy to take a closer look.
